prometheus/prometheus.go

```diff
@@ -0,0 +1,57 @@
+package prometheus
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type Prometheus struct {
+	duration *prometheus.HistogramVec
+	counter  *prometheus.CounterVec
+}
+
+type OptionFunc func(*Prometheus)
+
+func New(opts ...OptionFunc) *Prometheus {
+	labelNames := []string{"code", "handler", "method"}
+
+	m := &Prometheus{
+		counter: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "http_request_total",
+				Help: "A counter for http requests.",
+			},
+			labelNames,
+		),
+		duration: prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name:    "http_request_duration_seconds",
+				Help:    "A histogram of latencies for http requests.",
+				Buckets: []float64{.25, .5, 1, 2.5, 5, 10},
```

Comment:
Are these standard buckets that all services should use?

Should an option func be provided to allow the implementing service to customize the buckets?